src/notebook_unit1.ipynb

```diff
@@ -95,7 +95,7 @@
    "source": [
```

Comment:
<p>retruns</p><p><strong>No such file or directory: '../plots/expec_z.pdf</strong>'</p><p>i think it needs to create a plot folder first</p><p> </p><p>maybe at the start of the script already</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pauleonix/ssd_course/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>